src/SprykerEco/Zed/Braintree/Communication/Plugin/Checkout/BraintreePreCheckPlugin.php

```diff
@@ -16,11 +16,14 @@
 use SprykerEco\Shared\Braintree\BraintreeConfig;
 
 /**
+ * @deprecated Use {@link \SprykerEco\Zed\Braintree\Communication\Plugin\Checkout\BraintreeCheckoutPreConditionPlugin} instead.
+ *
  * @method \SprykerEco\Zed\Braintree\Business\BraintreeFacadeInterface getFacade()
  */
 class BraintreePreCheckPlugin extends BaseAbstractPlugin implements CheckoutPreConditionInterface
 {
     /**
+     * {@inheritDoc}
      * Specification:
```

Comment:
Please remove this line, example:
{@inheritDoc)
- Does 
- Does

Reply:
Updated.